### PR TITLE
release: jdk8u352-b08

### PR DESCRIPTION
**Pull request by zdtsw**
shipped:
x64 linux
ppc64le
aarch64
arm32

### CODE REVIEW

**Comment by gdams** [comment]
Testing end of code-freeze

**Comment by zdtsw** [comment]
not sure, seem GH action is not running on jdk8? @gdams

**Review by gdams — APPROVED** [review]

File: linux/jdk/debian/src/main/packaging/temurin/8/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+temurin-8-jdk (8.0.352.0.0+8-1) STABLE; urgency=medium
+
+  * Eclipse Temurin 8.0.352.0.0+8-1 release.
+
+ -- Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>  Tue, 25 Oct 2022 20:22:20 +0000
```
Comment (gdams) [inline linux/jdk/debian/src/main/packaging/temurin/8/debian/changelog:5]:
```suggestion
 -- Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>  Thu, 03 Nov 2022 20:22:20 +0000
```

For consistency with the RPMs

**Review by gdams — APPROVED** [review]

**Review by sxa — APPROVED** [review]
LGTM

**Pull request merged**